connectors/src/env.mjs

```diff
@@ -0,0 +1,82 @@
+import { z } from "zod";
```

Comment:
Not sure what this does - it came with the boilerplate.

Reply:
Let’s figure out

Reply:
Looks like it's important, so lets keep it: 
https://create.t3.gg/en/usage/env-variables

Reply:
looks like there are a bunch of stuff related to next JS in the file